docs/changelog/115834.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 115834
+summary: Try to simplify geometries that fail with `TopologyException`
+area: Geo
+type: bug
+issues: []
```

Comment:
Should we have a bug issue to close here, or does that not matter.

Reply:
I think it does not matter.